src/js/pickr.js

```diff
@@ -737,7 +737,7 @@ class Pickr {
         if (!this.options.disabled) {
             this._root.app.classList.add('visible');
             this._rePositioningPicker();
-            this._emit('show', this);
+            this._emit('show', this._color);
```

Comment:
After banging my head around with the plugin, I think this should be `this._lastColor` but I'm not sure, so please double check it.

Reply:
This is correct. `this._color` _is_ supposed to be the previous color but even I can't really wrap my head around anymore what's all happening in there 😅